website/docs/docs/dbt-versions/release-notes/04-Nov-2022/ide-features-ide-deprecation.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title: "Extra features in new IDE, and classic IDE deprecation"
```

Comment:
curious, are there version numbers we can use to talk about the new features and the old features? some specificity could be useful/helpful to people who are scanning the RNs on the DevHub site.

Reply:
good callout @nghi-ly ! hey @bossnunta - wondering if you know this re: version numbers?

Reply:
hey @nghi-ly - Nate has replied [here](https://dbt-labs.slack.com/archives/C043H8Z0P7H/p1669661560300659?thread_ts=1668813792.920259&cid=C043H8Z0P7H) 

> Unfortunately, we don't have version number for MT yet -- we do have that for ST though. These changes were ready with [v1.1.67.0](https://dbt-labs.slack.com/archives/C0164QWG2NA/p1668635992410339) for ST

Reply:
since we don't have versions for both st/mt - is it best to keep it as is? ideally, Nate would like the release note published this week